src/style.css

```
@@ -1 +1,57 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: inline-block;
+  margin: 10px 0 10px 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.nav__link:link {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-right: 20px;
+  padding: 24px 0;
+  color: #000;
+  border-bottom: 4px solid transparent;
+}
+
+.nav__link:visited {
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
+  padding-bottom: 20px;
+}
+
+.nav__link:last-child {
+  margin-right: 0;
+}
```

You will have `<li>`, and you can apply `:not(:last-child`), so it seems to be
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

---

is not fixed

---

Done